### ISSUE TITLE
SA1513 and SA1009 Conflict and code fixes fight between themselves

### ISSUE THREAD

**Issue by vbfox** [DotNetAnalyzers/StyleCopAnalyzers#1713]
In this program : 

```csharp
using System;

class Program
{
    private class Foo : IDisposable
    {
        public int Bar { get; set; }

        public void Dispose()
        {
        }
    }

    static void Main(string[] args)
    {
        foreach (
            var x in
            new[]
            {
                1,
                2,
                3
            })
        {
        }

        using (
            new Foo
            {
                Bar = 1
            })
        {
        }

        Console.WriteLine(
            new[]
            {
                1,
                2,
                3
            });
    }
}
```

Both the `foreach` and `using` trigger SA1513 at the end, but it isn't solvable, applying the codefix will produce SA1009.

Also applying SA1513 code fix once doesn't work, it add a line break between `}` and `)` but no new line, applying it a second time fixes it but SA1009 is still there.

**Comment by sharwell** [comment]
@vweijsters What did StyleCop classic do for this case?

**Comment by vweijsters** [comment]
SA1513 is not reported for the code above. After manually adding an empty line after the the closing brace, SA1009 is reported.

I would say that this should be corrected in SA1513, so that the code above will not raise SA1513.

**Comment by sharwell** [comment]
Agreed, thanks!

**Comment by vweijsters** [comment]
Grabbing this
